code/modules/cargo/bounties/special.dm

```diff
@@ -26,6 +26,13 @@
 	required_count = 10
 	wanted_types = list(/obj/item/stack/sheet/mineral/adamantine)
 
+/datum/bounty/item/trash
+	name = "Trash"
+	description = "Recently a group of janitors have run out of trash to clean up, without any trash Centcom wants to fire them to cut costs."
```

Comment:
most bounty descs have something like "Ship/Send/etc. X to Y"

```suggestion
	description = "Recently a group of janitors have run out of trash to clean up, without any trash Centcom wants to fire them to cut costs. Send a shipment of trash to keep them employed, and they'll give you a small compensation."
```

Reply:
done